### DISCUSSION TITLE
Allow to export to curl

### DISCUSSION THREAD

**Discussion by chalstrick** [thunderclient/thunder-client-support#615] [Ideas]
When working in a team I need to be able to show my requests and the results to colleagues. Therefore I would like to be able to export a single request with all the expanded parameters to curl syntax. Then I would execute the request on commandline and cut and paste request and response into a mail.

**Comment by rangav** [comment]
@chalstrick you can export to curl using code snippet menu

see details here
https://github.com/rangav/thunder-client-support#codegen

**Reply by chalstrick** [reply]
sorry, I overlooked that feature. Thanks
